stock_picking_invoice_link/models/sale_order.py

```diff
@@ -14,7 +14,8 @@ def get_stock_moves_link_invoice(self):
             lambda mv: (
                 mv.state == "done"
                 and not (
-                    any(
+                    self.env.context.get("check_invoice_state", True)
```

Comment:
This context check should be non required, but the contrary to abort on case of finding something.